docs/user-guide/configuration/parameters.md

```diff
@@ -9,14 +9,15 @@ toc:
       active: true
 ---
 ## Defining Parameters
-There are 2 ways to define parameters, you can see them both in the example below.
+There are 3 ways to define parameters, you can see them both in the example below.
 
 ```yaml
 parameters:
     nameA: "value1"
     nameB:
         value: "value2"
         description: "description of nameB"
+    nameC: ["value1", "value2"]
```

Comment:
should we add that the default will be the first value in the array?

Reply:
Yes, and It is important thing. I add about that.

Reply:
Hi @tkyi. Is it OK?

Reply:
small comment!